en/docs/learn/provisioning-tutorials.md

```diff
@@ -0,0 +1,8 @@
+# Provisioning 
+
+## Introduction
+
+Provisioning is the process of creating, maintaining and deleting digital identities (accounts) for users of a system(s) and linking appropriate rights to identities in the form of rules and roles. Identity provisioning is key for Identity Federation. Identity federation is a mechanism that allows authentication across different enterprises in different trust domains based on a trust factor. This makes access easy, as users do not have to remember a different set of credentials for every application they use. The following identity provisioning concepts are used in WSO2 Identity Server. 
```

Comment:
maintaining, and deleting

Reply:
Fixed in https://github.com/wso2/docs-is/pull/982 commit fa76065